sphinx/writers/latex.py

```diff
@@ -305,6 +305,11 @@ def escape_abbr(text: str) -> str:
 
 def rstdim_to_latexdim(width_str: str, scale: int = 100) -> str:
     """Convert `width_str` with rst length to LaTeX length."""
+    # MEMO: the percent unit is interpreted here as a percentage
+    # of \linewidth.  Let's keep in mind though that \linewidth
+    # is dynamic in LaTeX (e.g. smaller in lists), and may even
+    # be zero in some contexts (some table cells).  This is a legacy
```

Comment:
This will have to be corrected.  I apologize but it seems I made a confusion with some other LaTeX length, the `\baselineskip` which has nothing to do with `\linewidth`.  But it is true that `\linewidth` is a somewhat obscure beast in tabulary tables.